sdk_container/src/third_party/coreos-overlay/profiles/coreos/base/package.accept_keywords

```diff
@@ -14,6 +14,8 @@
 # Seems to be the only available ebuild in portage-stable right now.
 =app-crypt/adcli-0.9.2 ~amd64 ~arm64
 
+=app-crypt/ccid-1.5.4 ~arm64
```

Comment:
Please add a comment above it like `# Needed by arm64-native SDK.`.

Reply:
Done